src/lib/sleep.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Delays for input ms before resolving
+ */
+export default async function sleep(ms: number): Promise<void> {
+	return new Promise((resolve) => setTimeout(resolve, ms));
+}
```

Comment:
Just use the built in promisified `setTimeout`
```suggestion
import { promisify } from 'util';

const sleep = promisify(setTimeout);
export default sleep;
```
And if you update to nodejs 16 you can do
```suggestion
export { setTimeout as sleep } from 'timers/promises';

Reply:
Also I'd probably name it just `setTimeout` since it's less confusing when using the native apis